app/javascript/styles/calls.scss

```diff
@@ -0,0 +1,27 @@
+// Styling related to log-a-call modals.
+
+@import './_variables';
+
+// Core new call modal
+.calls-layout {
```

Comment:
A soft refactor here to nest these appropriately, and take out the base html elements (a bunch of them were `h4.whatever`, and that's not necessary since it's overruling a bunch of those styles).